### PR TITLE
Issue 15 - Create general views/templates for Backend store

### LINKED ISSUE #15: Create general views/templates for Backend store
General views and templates should be added to the project:
- index page with template
- common header
- common footer
- common container layout

### PR DESCRIPTION
**Pull request by sshishov**
Added general template directory and per app directory
Updated URLs to incude index pages for the project and store application
Added `index_view`s for project and store application
The following hierarchy is used:

```
  base.html <- main.html <- index.html (project)
  base.html <- main.html <- store/main.html <- store/index.html (store)
```

Added `header` and `footer` placeholders
Added appropriate placeholders to index pages

Fixes #15 

@zitoss please review and understand changes

### CODE REVIEW

File: avtozip/avtozip/settings.py [hunk 1]
```diff
@@ -56,10 +56,13 @@
 
 ROOT_URLCONF = 'avtozip.urls'
 
+print(BASE_DIR)
```
Comment (zitoss) [inline avtozip/avtozip/settings.py]: I think it should be deleted. Please check.

Comment (sshishov) [inline avtozip/avtozip/settings.py]: You are right, our code should not have `print` statements at all. It was debug purpose

**Comment by zitoss** [comment]
Please rebase to latest master!

**Comment by zitoss** [comment]
The code seems OK, please merge to master ;-)

**Pull request merged**